doc/ipython-notebooks/data_project/health_data_project.ipynb

```diff
@@ -0,0 +1,2248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Applying machine learning algorithms on MIMIC\n",
+    "\n",
+    "## Patient Monitoring and Decision Support using Health Data\n",
+    "\n",
+    "**Notebook by Olivier Nguyen (Github: [olinguyen](http://github.com/olinguyen)) for GSoC 2017**\n",
+    "\n",
+    "**Mentors: [Lea Goetz](https://github.com/lgoetz), [Heiko Strathmann](https://github.com/karlnapf)**\n",
+    "\n",
+    "### Introduction\n",
+    "\n",
+    "Everyday, the healthcare industry creates large amounts of patient and clinical data and stores them in electronic health records. Most of this data has previously been inaccessible, in part due to patient privacy concerns, which poses a challenge to researchers working on the analysis of health records.\n",
+    "\n",
+    "However, initatives like the Medical Information Mart For Intensive Care (MIMIC) database project have allowed for everyone to use and experiment with health data. In particular, the MIMIC database is a critical care database made freely available for researchers around the world to develop and evaluate intensive care unit (ICU) patient monitoring and decision support systems that will improve the efficiency, accuracy and timeliness of clinical decision-making in critical care.\n",
+    "\n",
+    "### Objective\n",
+    "\n",
+    "In the [previous notebook](https://github.com/olinguyen/gsoc2017-shogun-dataproject/blob/master/Exploring MIMIC.ipynb), we were introduced to the MIMIC database to better understand the dataset and visualize its contents. In this notebook, we'll dive into the data analysis component of this project. Using the MIMIC database which includes vital signs, medications, diagnostic code and many more variables, along with feature engineering techniques, I investigated whether I could build a robust classifier to perform the following tasks which can be used to help hospitals and care units make decisions about the management of resources and staffing:\n",
```

Comment:
We usually don't put link to external notebooks in ours. 
Maybe we can either remove this, or merge some parts of it?